### ISSUE TITLE
Modifiers / Tracking Speed For All

### ISSUE THREAD

**Issue by viviengarcia** [linearmouse/linearmouse#813]
### OS

macOS 14.6.1

### LinearMouse

v0.10.0

### Is your feature request related to a problem?

I have a bit of a hard time with Studio, the official Lego 3D construction app. It maps camera view to right-click drag, and the sensitivity is too much and not customizable. I need to have a different tracking speed when I right-click drag from when I move the cursor normally.

### Describe the solution you'd like

I know there is already a similar feature request but I think mine goes further and would require a bit more in the UX department, hence this issue.

Basically, each interaction from the mouse (moving cursor, clicking, click-dragging and scrolling) should benefit from sensitivity adjustment and modifiers options. It's already nearly there, with scrolling having everything, but Pointer doesn't have modifiers and Buttons don't have modifiers not tracking speed.

### Additional context

Not really additional context, but I find your app awesome already and wanted to thank you far it. Can't do much more than that unfortunately (perhaps French translation? I don't know).
